ophyd/_pyepics_shim.py

```diff
@@ -19,8 +20,8 @@
 
 
 def get_pv(*args, **kwargs):
-    import epics
-    kwargs.setdefault('context', epics.ca.current_context())
+    if not LooseVersion(epics.__version__) >= LooseVersion('3.4.0'):
```

Comment:
Minor readability improvement by avoiding the double-negative.

```suggestion
    if LooseVersion(epics.__version__) < LooseVersion('3.4.0'):
```

Reply:
Might be worth linking, in a comment, to this commit where the change we are accounting for here was introduced. https://github.com/pyepics/pyepics/commit/1ca46afa5afe285fecd4ea1d12a7eeee0376c773

h/t @tacaswell for digging that up.